rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,133 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# TL Quarterly Planning Process
+
+## Summary
+
+This document details the a quarterly process by which TLs will contribute to the overall DCP quarterly roadmapping
+process via a Technical Objectives RFC, presented during the TL+PM Planning Call.
+
+## Author(s)
+
+[Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+## Motivation
+
+The new Roadmaps and Planning Process, detailed in the RFC here,  defines a process for annual roadmaps that inform
```

Comment:
Are you referring to the broader planning RFC here?

Reply:
Oops forgot to actually add the link 🤦